hnn_core/viz.py

```diff
@@ -110,7 +110,7 @@ def plot_dipole(dpl, tmin=None, tmax=None, ax=None, layer='agg', decim=None,
 
     ax.ticklabel_format(axis='both', scilimits=(-2, 3))
     ax.set_xlabel('Time (ms)')
-    ax.set_ylabel('Dipole moment')
+    ax.set_ylabel('Dipole moment (nAm)')
```

Comment:
I guess this was already implemented in #250 , need a rebase?

Reply:
[Doesn't seem so](https://github.com/jonescompneurolab/hnn-core/blob/master/hnn_core/viz.py#L113)